expressions.js

```diff
@@ -102,4 +106,20 @@ module.exports = {
       return `${LEGACY_CATALOG_URL}${match.input}${reconstructQuery(query)}`
     }
   },
+  /**
+   *  Handle requests on //browse.nypl.org/iii/encore/logoutFilterRedirect
+   *  Emulate Encore behavior by redirecting to the CAS logout endpoint,
+   *  passing either the given service= param or Vega Home.
+   *  (The service= param controls where CAS redirects to after logout.)
+   */
+  encoreLogoutFilterRedirect: {
+    expr: /^\/iii\/encore\/logoutFilterRedirect\b/,
+    handler: (match, query) => {
+      const redirectUri = Array.isArray(query.redirect_uri) ? query.redirect_uri[0] : null
```

Comment:
where does the redirect_uri param come from?

Reply:
Nowhere yet, but an update to our Login server is coming that will pass a redirect_uri= param to the Encore Logout route. Encore itself will ignore it, but once the RedirectService is handling old Encore requests, we'd like the RedirectService to honor it.